import tensorflow


#It actually is not an error but rather a warning sign I guess. That why it still could execute.